### PR TITLE
Modifications to improve Pjax scroll handling

### LINKED ISSUE #35: Improve Pjax scroll handling

### PR DESCRIPTION
**Pull request by anita-chouhan**
Fixes #35

### CODE REVIEW

File: assets/js/pjaxinit.js [hunk 1]
```diff
@@ -20,6 +21,16 @@
       },
     },
     cacheBust: false,
+    scrollRestoration: false,
+    scrollTo: disableScroll === true ? false : 0,
+    analytics: function () {
```
Comment (rungta) [inline assets/js/pjaxinit.js]: No need for this. GA4 automatically tracks Pjax visits, right?

File: assets/js/pjaxinit.js [hunk 2]
```diff
@@ -47,4 +83,13 @@
   });
   document.addEventListener('pjax:send', topbar.show);
   document.addEventListener('pjax:complete', topbar.hide);
+
+  // Add Pjax to content inserted by HTMX
+  document.addEventListener('htmx:afterSwap', function (htmxEvent) {
```
Comment (rungta) [inline assets/js/pjaxinit.js]: Why are we adding HTMX related code in this pull request?

**Review by rungta — CHANGES_REQUESTED** [review]

File: assets/js/pjaxinit.js (hunk 1 above)
Comment (anita-chouhan) [reply assets/js/pjaxinit.js]: Removed unused GA4 tracking as it's not needed. GA4 automatically tracks Pjax visits.

File: assets/js/pjaxinit.js (hunk 2 above)
Comment (anita-chouhan) [reply assets/js/pjaxinit.js]: Removed as HTMX is not in our boilerplate. Initially, I thought it would be beneficial as we are improving Pjax.

**Review by rungta — APPROVED** [review]

**Review by rungta — APPROVED** [review]

File: assets/js/pjaxinit.js (hunk 2 above)
Comment (rungta) [reply assets/js/pjaxinit.js]: We were improving the _scroll handling_ of Pjax, so only scroll related changes should be part of this pull requests.

Comment (rungta) [reply assets/js/pjaxinit.js]: We were improving the _scroll handling_ of Pjax, so only scroll related changes should be part of this pull request.

**Pull request merged**